index.js

```diff
@@ -1,508 +0,0 @@
-    }
-  } else if (
-    answer.toLowerCase().includes("examine") ||
-    answer.toLowerCase().includes("look at") ||
-    answer.toLowerCase().includes("inspect") ||
-    answer.toLowerCase().includes("view")
-  ) {
-    lookupTable[answer].examine();
-    return gamePlay();
-  } else if (answer.toLowerCase().includes("drop")) {
-    lookupTable[answer].drop();
-    return gamePlay();
-  } else if (answer === "xyzzy") {
-    //cheat code
-    console.log(
-      "You entered the cheat code! Unfortunately in this game that means you die!!!....."
-    );
-    process.exit();
-  } else if (answer.toLowerCase().includes("inventory")) {
-    showInventory();
-    return gamePlay();
-  } else if (
-    answer.toLowerCase() === "open door" ||
-    answer.toLowerCase() === "use key" ||
-    answer.toLowerCase() === "unlock door"
-  ) {
-    if (player.inventory.includes(keyRoomOne)) {
-      player.inventory.pop();
-      changeRoom("roomTwo");
-      console.log(`You're now in Room 2!\n${roomTwo.description}`);
-      return gamePlay();
-    } else {
-      console.log("The door is locked!");
-      return gamePlay();
-    }
-  } else if (answer.toLowerCase() === "use keypad") {
-    let canOpen = await useKeyPad();
-    if (canOpen) {
-      changeRoom("roomThree");
-      console.log(`You're now in room 3!\n${roomThree.description}`);
-    } else {
-      console.log("Wrong combo. Try again.");
-    }
-    return gamePlay();
-  } else if (
-    answer.toLowerCase() === "throw rock" ||
-    answer.toLowerCase() === "use rock"
-  ) {
-    if (player.inventory.includes(rock) && player.currentRoom === "roomThree") {
-      player.inventory.pop();
-      changeRoom("roomFour");
-      console.log(
-        `You smashed the window and crawled through.\nYou have entered Room 4!\n${roomFour.description}`
-      );
-      return gamePlay();
-    }
-  } else if (
-    answer.toLowerCase() === "use snickers bar" ||
-    answer.toLowerCase() === "feed ogre"
-  ) {
-    if (
-      player.inventory.includes(snickers) &&
-      player.currentRoom === "roomFour"
-    ) {
-      player.inventory.pop();
-      changeRoom("roomFive");
-      console.log(
-        `You've fed the Ogre.\nNow that his hunger has been satisfied he opens the door for you and you enter room 5!\n${roomFive.description}`
-      );
-      return gamePlay();
-    } else {
-      console.log(
-        "You have nothing to feed the ogre! Find something quick before he eats you!"
-      );
-      return gamePlay();
-    }
-  } else if (answer.toLowerCase() === "use molotov cocktail") {
-    console.log(
-      "You burned down the building and died inside... Better luck next time."
-    );
-    process.exit();
-  } else if (answer.toLowerCase() === "use axe") {
-    console.log(
-      "You broke down the door and escaped!! Congratulations you escaped!"
-    );
-    process.exit();
-  } else {
-    console.log("Invalid input. Try again!");
-    return gamePlay();
-  }
-}
```

Comment:
I'm sure you would agree with my saying the logic for the gameplay is a bit convoluted still.  Definitely better: I got through it! Some of the nested `if` statements cause issues with the error catching.  For example, when you finally get to the ogre in room 4, invalid input will throw an input rather than `console.log("Invalid input. Try again!")`, which is properly implemented elsewhere.  This is due mostly to nested logic blocks. If your commands were on the same level of scope, the `else` block on line 504 will ensure no errors, because it will catch every input that isn't satisfied by the previous conditions.  But when you have logic blocks in different scopes, you have to put out more and more fires, and like the `lookupTable`, just invite more room for error.